app/partnership/page.tsx

```diff
@@ -117,28 +139,83 @@ export default function Component() {
           </div>
         </div>
       </header>
-      <section className="py-12 md:py-16 lg:py-20">
-        <div className="container px-4 md:px-6">
-          <h2 className="text-4xl font-bold mb-8">Our Sponsors</h2>
-          <div className="grid grid-cols-2 sm:grid-cols-3 md:grid-cols-4 lg:grid-cols-5 gap-6 md:gap-8">
-            {sponsors.map((partner) => (
-              <div
-                key={partner.name}
-                className="flex justify-center items-center"
-              >
-                <Image
-                  src={partner.logo}
-                  alt={partner.name}
-                  width={200}
-                  height={200}
-                />
-              </div>
-            ))}
+
+      <div className="container flex flex-col gap-5 mt-10 lg:w-1/2 text-center justify-center items-center ">
+        <h2 className="text-4xl font-bold mt-8 text-center">Our Sponsors</h2>
+        <p className="text-gray-600 dark:text-gray-400">Becoming a sponsor for ICONIC 2024, a biennial international scientific conference in Germany by PPI Jerman, offers your company a prime opportunity to enhance its visibility on the international academic stage. ICONIC bridges Indonesia and Germany through discussions on sustainability and environmental awareness, engaging key decision-makers and industry leaders from academia, economics, and policymaking.</p>
+        <p className="text-gray-600 dark:text-gray-400">By supporting ICONIC 2024, you can effectively reach your target audience, build meaningful connections, and demonstrate your commitment to sustainable solutions. Seize this chance to connect, influence, and lead towards a more inclusive and sustainable future.</p>
+        <div className="flex sm:flex-row flex-col gap-5 mt-5">
+          <div className="sm:w-fit flex p-5 pe-0.5 justify-between sm:justify-center bg-[#66d998] rounded shadow-lg">
```

Comment:
mel, ini lebih enak jadi bg-gray-100 atau bg-gray-200

```suggestion
          <div className="sm:w-fit flex p-5 pe-0.5 justify-between sm:justify-center bg-gray-100 rounded shadow-lg">
```

atau 

```suggestion
          <div className="sm:w-fit flex p-5 pe-0.5 justify-between sm:justify-center bg-gray-200 rounded shadow-lg">
```

Reply:
btw kalo warna teks nya jadi gak keliatan mungkin bisa agak gelapan, make sure keliatan ajaa sii.

Ato preferably pake accent2 atau secondary

Reply:
soalnya logo ecadinnya warna putih ca